apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/variables/index.vue

```diff
@@ -64,12 +70,80 @@ const pageFilterTemplate: IFilter[] = [
       ontologyTableId: "Keywords",
       ontologySchema: "CatalogueOntologies",
       columnId: "keywords",
-      initialCollapsed: false,
+      initialCollapsed: true,
     },
     conditions: [],
   },
+  {
+    id: "cohorts",
+    config: {
+      label: "Cohorts",
+      type: "REF_ARRAY",
+      refTableId: "Cohorts",
+      buildFilterFunction: (
+        filterBuilder: Record<string, Record<string, any>>,
+        conditions: IFilterCondition[]
+      ) => {
+        return {
+          ...filterBuilder,
+          ...{
+            _or: [
+              {
+                mappings: {
+                  source: { equals: conditions.map((c) => ({ id: c.name })) },
```

Comment:
I believe this should also filter on match = 'partial' or  match = 'complete' (e.g. this data https://preview-emx2-pr-3656.dev.molgenis.org/catalogue-demo/tables/#/VariableMappings?_filter=match&match=[{%22name%22:%22complete%22},{%22name%22:%22partial%22}])

Reply:
solution is to add `match: { name: { equals: [ "complete" ,"partial"] }} `